packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -37,10 +37,16 @@ class CreateCommand extends FlutterCommand {
       help: 'Also add a flutter_driver dependency and generate a sample \'flutter drive\' test.'
     );
     argParser.addFlag(
-      'plugin',
+      'package',
```

Comment:
The first rest argument is currently used for the output location: `flutter create <output directory>`, so I think it'd be problematic to use it for the project type as well.

> I personally think having the shorter flags is cleaner and easier to use than the longer --template=...

Gotcha. The length may not be critical here, since flutter create a less common command (compared to something like `flutter run`). I think I still prefer the option flag (`--template=foo`) - I do think it makes it more clear that you're selecting between a few different project templates.